lib/codecs/CryptosuiteDecoder.js

```diff
@@ -18,8 +17,7 @@ export class CryptosuiteDecoder extends CborldDecoder {
     }
```

Comment:
Hmm, should this value check be done in `createDecoder` below instead ... and return `false` down there if the value type isn't a `number`? Wouldn't that be more consistent? (I realize this isn't code that was touched in this PR previously, just seems inconsistent here).

Reply:
Oh, I see a comment below that the CryptosuiteEncoder isn't used ... does that mean this decoder isn't used either? If so, let's remove them.